src/Hooks/useStyles.tsx

```diff
@@ -1,4 +1,4 @@
-import { makeStyles, createStyles } from '@material-ui/core/styles';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
```

Comment:
Are any of the styles in this file being used? If they are they should be broken out and put adjacent to the components